core/src/templates/components/global-footer/global-footer.twig

```diff
@@ -14,9 +14,9 @@
 
 <section class="su-global-footer {{ modifier_class }}">
```

Comment:
Hm... good thinking, @cjwest, but I"m thinking it might be a little easier in this case to change the outer `section` tag to a simple `div` tag, since in some variations of the local footer, the logo lockup is option.

Reply:
Awww, you're right. That goes for both of these. 😞

Reply:
Actually....I just realized that this is for the Global Footer - so maybe it is ok 🤔 , but, the `div` change would likely be more straightforward since that likely won't involve CSS changes, while making the logo change to `h2` might (since h2 might or might not have some styles that need to be overridden). I know there are some debates on whether it is good practice to make logo links a heading - so maybe something worth looking into also 😃

Reply:
Here's a semi-related post: https://www.fastcompany.com/3016894/should-your-tag-be-your-logo
By making the logo a heading, it gets called out for assistive technologies and could make the page easier to navigate.

Reply:
@yvonnetangsu,
Thanks for considering this. When running through with the screen reader, I figure that the `<h2>` isn't really necessary. The logo already shows up as a link, and the global navigation shows up. That should work.